Rakefile

```diff
@@ -11,7 +11,7 @@ task :console do
   if File.exist? config_file
     config = YAML.load File.read(config_file)
 
-    AWS.config(config['aws'])
+    Aws.config = config['aws']
```

Comment:
This isn't particularly to do with this PR, but it's convenient to bring it up here for discussion. What if I need to use some ruby code in my AWS config? For example, right now I need to use `logger: Logger.new($stdout)`, and I can't just put that in my YAML config file.

@phstc thoughts?